src/components/Network.js

```diff
@@ -103,15 +125,9 @@ class Network extends React.Component {
     };
 
     const renderConnected = (peer, conn) => {
-        const sync = conn.plugins["node-sync-timestamp"];
-        const first_timestamp = this.state.dag.first_timestamp;
-        const latest_timestamp = this.state.dag.latest_timestamp;
-        const delta = latest_timestamp - first_timestamp;
-        const synced_percent = 100 * (sync.synced_timestamp - first_timestamp) / delta;
-        let general_percent = (100 * (sync.latest_timestamp - first_timestamp) / delta) - synced_percent;
-        if (general_percent > 100) {
-          general_percent = 100;
-        }
+        const sync_v1 = conn.plugins["node-sync-timestamp"];
+        const sync_v2 = conn.sync;
+        const { synced_percent, general_percent } = this.getSyncProgress(sync_v1, sync_v2);
```

Comment:
You can add a `sync_version` variable that will be used later. What do you think?

Reply:
It will not be required if we adopt the new `sync.data` suggested [here](https://github.com/HathorNetwork/hathor-explorer/pull/42#discussion_r600788906).

Reply:
Done